### DISCUSSION TITLE
Ryujinx Compatibility

### DISCUSSION THREAD

**Discussion by GamerNinja82** [Nifyr/Imposters-Ordeal#21] [Q&A] [answered]
Hello,
I was wondering if anyone here knows how to get this working on ryujinx because i cant seem to get the romfs files to be detected

**Comment by Nifyr** [comment, marked as answer]
Should be as simple as putting IO's output into Ryujinx's mod directory as per usual LayeredFS procedure.
1. Right click BD/SP in the Ryujinx app and select "Open Mods Directory".
2. Drag the Output folder from IO into this directory. You can also rename this folder if you want to.
3. Launch BD/SP in Ryujinx like normal.
![image](https://github.com/Nifyr/Imposters-Ordeal/assets/34029571/55a0292d-032b-4f01-9d6f-2d8476043cdc)
![image](https://github.com/Nifyr/Imposters-Ordeal/assets/34029571/deb26d35-b345-4bc6-a5a2-317a9fe212f2)

**Reply by GamerNinja82** [reply]
Okay and so how would i pair this with luminescent Platinum just have them both in there? Also i meant I can’t seem to get imposters ordeal to find the romfs folder to work

**Reply by Nifyr** [reply]
IO expects the game dump of BD/SP to look something like this:
![image](https://github.com/Nifyr/Imposters-Ordeal/assets/34029571/f6f1df66-7907-4c02-a38a-efa104a6bd67)
(this is a dump from yuzu, but Ryujinx should output the same)
Select the folder containing romfs when prompted for a dump upon launching IO. ("0100000011D90000" in the picture above, but it doesn't need to have that name) Importantly, the dump's folder structure needs to match what's expected by IO as shown above.

Pairing LayeredFS mods in the way you suggested often leads to issues with overwriting changes. If you wish to combine the use of IO with other mods, there's a button for that once you've loaded a dump:
![image](https://github.com/Nifyr/Imposters-Ordeal/assets/34029571/4e8feb6c-24bf-45d9-af1e-8ca6d3a32de6)
This will update editors to display and edit any relevant data from the selected mod, as well as include the mod in the output when exporting.
